Celeste.Mod.mm/Patches/Monocle/Tracker.cs

```
@@ -136,15 +136,11 @@ public static void AddTypeToTracker(Type type, Type trackedAs = null, params Typ
             }
         }
 
-        private static bool AddSpecificType(Type type, Type trackedAsType, Dictionary<Type, List<Type>> tracked, HashSet<Type> knownTypes) {
+        private static bool AddSpecificType(Type type, Type trackedAsType, Dictionary<Type, List<Type>> tracked) {
             if (type.IsAbstract) {
                 return false;
             }
 
-            // Make sure the tracker knows about both the type and trackedAs type, to fix scenarios like `[TrackedAs(typeof(UntrackedType))]`
```

Please leave the comment, since it is still relevant, but move it to the right place.

---

If a mod uses `TrackedAs` and then tries `GetEntities<OriginalClass>`, that's already incorrect usage. I think we should just let it crash here rather than hide the bug

---

I agree, I'm mostly worried about backwards compat.